tests/test_models.py

```diff
@@ -125,3 +126,17 @@ def test_cons_replacing():
         assert True is False
     except IndexError:
         pass
+
+
+def test_number_model_copy():
+    i = HyFloat(42)
```

Comment:
Did you mean to say `HyInteger` here? There's a `HyFloat` test just a few lines down.